src/Components/Form/FormFields/FormField.tsx

```diff
@@ -71,7 +71,18 @@ const FormField = ({
         )}
       </div>
       <div className={field?.className}>{children}</div>
-      <FieldErrorText error={field?.error} className={field?.errorClassName} />
+
+      {field?.error && (
+        <ul>
+          {(typeof field.error === "string" ? field.error : String(field.error))
+            .split("\n")
+            .map((err, index) => (
+              <li key={index}>
+                <FieldErrorText error={err} className={field?.errorClassName} />
+              </li>
+            ))}
+        </ul>
```

Comment:
This change broke the spacing in the entire platform...


<img width="1652" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/69d6fdc9-c188-40b6-a078-29151bb85284">

Reply:
![image](https://github.com/coronasafe/care_fe/assets/91010142/271c620d-3a9f-4280-ad81-92752a086a81)

It is because I'm checking before rendering
but after removing that condition, it fixed
![image](https://github.com/coronasafe/care_fe/assets/91010142/5a9b82ff-7ba5-4f4b-b633-ad4b8eb5581e)
![image](https://github.com/coronasafe/care_fe/assets/91010142/04e3def0-3a15-414f-aece-11efb18944ce)